docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| `authorityInformationAccess`      | SHOULD          | N                     | See [Section 7.1.2.8.2](#71282-authority-information-access) |
+| `nameConstraints`                 | MAY             | \*[^name_constraints] | See [Section 7.1.2.8.7](#71287-name-constraints) |
+| Signed Certificate Timestamp List | MAY          | N                | See [Section 7.1.2.9.4](#71294-signed-certificate-timestamp-list) |
+| Any other extension               | SHOULD NOT   | -                     | __**TBD**__ |
+
+Table: Extensions when the Subordinate CA is operated by an entity that is not the Issuing CA or an Affiliate of the Issuing CA.
+
+| __Extension__                     | __Presence__  | __Critical__          | __Description__ |
+| ----                              | -             | -                     | ----- |
+| `authorityKeyIdentifier`          | MUST          | N                     | See [Section 7.1.2.9.2](#71292-authority-key-identifier) |
+| `basicConstraints`                | MUST          | Y                     | See [Section 7.1.2.8.3](#71283-basic-constraints) |
+| `certificatePolicies`             | MUST          | N                     | See [Section 7.1.2.8.5](#71285-certificate-policies---non-affiliated-ca) (Non-Affiliated CA) |
+| `crlDistributionPoints`           | MUST          | N                     | See [Section 7.1.2.9.3](#71293-crl-distribution-points) |
+| `keyUsage`                        | MUST          | Y                     | See [Section 7.1.2.8.6](#71286-key-usage) |
+| `subjectKeyIdentifier`            | MUST          | N                     | See [Section 7.1.2.9.5](#71295-subject-key-identifier) |
+| `extKeyUsage`                     | MUST[^eku_ca] | -                     | See [Section 7.1.2.2.4](#71224-extended-key-usage---restricted-cross-certified-ca) |
+| `authorityInformationAccess`      | SHOULD        | N                     | See [Section 7.1.2.8.2](#71282-authority-information-access) |
+| `nameConstraints`                 | MAY           | \*[^name_constraints] | See [Section 7.1.2.8.7](#71287-name-constraints) |
+| Signed Certificate Timestamp List | MAY           | N                     | See [Section 7.1.2.9.4](#71294-signed-certificate-timestamp-list) |
+| Any other extension               | SHOULD NOT   | -                     | __**TBD**__ |
+
+[^eku_ca]: While [RFC 5280, Section 4.2.1.12](https://tools.ietf.org/html/rfc5280#section-4.2.1.13) notes that this extension will generally only appear within end-entity certificates, these Requirements make use of this extension to further protect relying parties by limiting the scope of CA Certificates, as implemented by a number of Application Software Suppliers.
+
+[^name_constraints]: See [Section 7.1.2.8.7](#71287-name-constraints) for further requirements, including regarding criticality of this extension.
```

Comment:
So, I think your comment is worded very confusing, and I think you've chosen a very weird section to comment.

There are two sections for nameConstraints already - 7.1.2.4.2 and 7.1.2.8.7.

I think your point that you were trying to convey is 7.1.2.8.7 should list MAY instead of MUST. Is that correct?

Reply:
Yes, I was reading those two sections independently, and thought what would happen if a CA wanted to use nameConstraints but not fully technically constrain per 7.1.2.4.2. Switching MUST to MAY in 7.1.2.8.7 seems to work but you could also start the paragraph with "For Technically Constrained subCAs per section 7.1.2.4, the Name Constraints extension MUST be encoded...."